### ISSUE TITLE
Length of EQU name limit?

### ISSUE THREAD

**Issue by richardturnnidge** [envenomator/agon-ez80asm#102]
Is there a limit on chars in an EQU?

Trying to debug someone else's code and the build fails error 5 trap, or something like that. Sounds like too much trying to get into too little space.

If so, just add name limit to docs??

May just be similar to file name lengths issues we had.

UPDATE. Seems to fail if more than 32 chars. Feels like a suspicious number!! ;-)

This works:
VDU_5678901234567890123456789012:  equ 136

This does not:
VDU_56789012345678901234567890123:  equ 136

./build.sh: line 24: 62385 Trace/BPT trap: 5       /usr/local/bin/ez80asmm4 "$1" "-l" "-x"

**Comment by envenomator** [comment]
There is a maximum limit of 32 characters on a label. I'll put it in the docs - https://github.com/envenomator/agon-ez80asm/commit/b0a7bcedf6bd31b9a9e6440aecf51457ea4617da

**Comment by richardturnnidge** [comment]
Good call. Glad that was the likely culprit :-)

I guess that is ANY label?

BTW, I think I may have had this issue myself ages ago, only just coming back into my memory!!!

**Comment by envenomator** [comment]
Any global label indeed; I'll need to check the size maximums for local labels after the @ next

**Comment by envenomator** [comment]
Yes, the same, though technically they might work with longer names depending on their scope. However, supported up to and including 32 characters, updated the docs

**Comment by envenomator** [comment]
These are BIG-ass labels ;-)

**Comment by richardturnnidge** [comment]
He was using things like this:

VDU_SCREENMODE_640x240x16_60HZ_DB:  equ 132

which is big, but understandable.

**Comment by richardturnnidge** [comment]
Totally unrelated, but as I have no other way to contact you anymore, just wondering if you might have any thoughts on my i2c issues and if my thoughts on this 'bug' makes sense???

https://github.com/AgonConsole8/agon-mos/issues/118

Not expecting you to fix it, just see if my idea/suspicions seems logical.

**Comment by envenomator** [comment]
Released as v2.0
